README.md

```diff
@@ -26,10 +26,11 @@ $ npm install serverless -g
 $ npm install
 
 # Get a bot from Telegram, sending this message to @BotFather
+Go to https://t.me/BotFather
 $ /newbot
 
 # Put the token received into a file called serverless.env.yml, like this
-$ cat serverless.env.yml
+$ cat >serverless.env.yml
```

Comment:
Why added `>`

Reply:
actually in ubuntu cat filename doesn't works for creating new file, it is used for viewing existing new file, cat >filename creates new file and takes input along with it to be put into file.

Reply:
https://www.geeksforgeeks.org/cat-command-linux-examples/